Content.Client/Resources/ResourceTypes/ResourceSound.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using System.IO;
+using Robust.Client.Audio;
+using Robust.Client.Graphics;
+using Robust.Shared.Audio;
+using Robust.Shared.IoC;
+
+namespace Content.Client.Resources.ResourceTypes {
+    public class ResourceSound : DreamResource {
+        private readonly AudioStream _stream;
+
+        public ResourceSound(string resourcePath, byte[] data) : base(resourcePath, data) {
+            if (resourcePath.EndsWith(".ogg"))
+                _stream = IoCManager.Resolve<IClydeAudio>().LoadAudioOggVorbis(new MemoryStream(data), resourcePath);
+            else if (resourcePath.EndsWith(".wav"))
+                _stream = IoCManager.Resolve<IClydeAudio>().LoadAudioWav(new MemoryStream(data), resourcePath);
+            else
+                throw new Exception("Only *.ogg and *.wav audio files are supported.");
```

Comment:
If merged, should document this on the wiki page or whatever as a difference, since BYOND supports mp3, midi, and all those keygen music formats.

Reply:
I mean, that's *for now*. We can always add those later. It's not like OpenDream supports them currently does it?